[Github] Reformat strings for code format action

boomanaiden154 opened the pull request:
Before this patch, there was a regression in comment formatting due to some code formatting in bd3e8eb6e325081bf7cfbe93652aa825de3170e5. This was fixed in 428660cfb986dd0a59cd2a16972c5f7109080522. Github interprets a tab before a string as starting code formatting. The message that indicted the code formatting in a PR had been fixed was refactored to a python multi-line string, but with a tab in front, causing these messages to be rendered as code blocks in Github, instead of as intended. This patch builds upon the original fix to reformat the strings so that they fit within ~80 character lines and are simpler to modify in the future, hopefully removing traps like the one that caused the original issue.

boomanaiden154 committed:
[Github] Fix comment formatting for code format action

Before this patch, there was a regression in comment formatting due to
some code formatting in bd3e8eb6e325081bf7cfbe93652aa825de3170e5. Github
interprets a tab before a string as starting code formatting. The
message that indicted the code formatting in a PR had been fixed was
refactored to a python multi-line string, but with a tab in front,
causing these messages to be rendered as code blocks in Github, instead
of as intended. This patch changes this back to a single line string to
prevent this issue.

boomanaiden154 commented:
If reviewers prefer, we could also emulate a multi-line string without the mentioned drawback by using string concatenation:
```python
test = ("This is a test. "
        "This is another sentence in the same string.")

print(test)
```

Which produces the following (as expected):
```
This is a test. This is another sentence in the same string.
```

tru commented:
Aha! I was wondering why the emojis didn't work anymore. Thanks for fixing it. I think concat with () is a little nicer so let's use that.

DavidSpickett commented:
I fixed it another way with https://github.com/llvm/llvm-project/commit/428660cfb986dd0a59cd2a16972c5f7109080522, but your change also removes an extra newline so that's nice improvement.

(the newline was unrelated to the problem)

boomanaiden154 commented:
> I fixed it another way with [428660c](https://github.com/llvm/llvm-project/commit/428660cfb986dd0a59cd2a16972c5f7109080522), but your change also removes an extra newline so that's nice improvement.

I've updated the patch to fix formatting there and in two other places. Thoughts?

tru reviewed (approved):
LGTM!
